docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -306,6 +306,37 @@ Possible values are:
 
 The password to connect to the slashing protection database.
 
+#### `slashing-protection-db-pool-configuration-file`
+
+=== "Syntax"
+
+    ```bash
+    --slashing-protection-db-pool-configuration-file=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --slashing-protection-db-pool-configuration-file=/Users/me/config/HikariConfig
```

Comment:
Lets add .properties extension to make the intent clear that we are talking about a properties file.
`--slashing-protection-db-pool-configuration-file=/Users/me/config/HikariConfig.properties`